components/cloud-foundry/frontend/i18n/en/app.json

```diff
@@ -19,21 +19,332 @@
+            "endpoint-label": "[[@:endpoint]]",
+            "org-label": "@:cf.organisation.organisation",
+            "space-label": "@:cf.space.space",
+            "edit-button": "Edit",
+            "edit-dialog": {
+              "title": "Edit App",
+              "name-label": "Application Name",
+              "name-error-required": "Name is required",
+              "memory-label": "Memory Usage",
+              "memory-error-required": "Memory is required",
+              "memory-error-min": "Memory must be higher than 0",
+              "instances-label": "Instances",
+              "instances-error-required": "Instances is required",
+              "instances-error-min": "Instances must be 0 or more",
+              "instances-error-whole": "Instances must be whole number"
+            }
+          },
+          "instances-panel": {
+            "title": "Instances",
+            "none": "This application has no running instances.",
+            "index-label": "Index",
+            "status-label": "Status",
+            "memory-label": "Memory",
+            "disk-label": "Disk",
+            "uptime-label": "Uptime",
+            "action-label": "Action",
+            "show-all": "Show All"
+          },
+          "routes-panel": {
+            "title": "Routes",
+            "add-button": "Add route",
+            "none": "You have no routes.",
+            "route-label": "Route",
+            "action-label": "Action",
+            "route-actions": {
+              "unmap": "Unmap from App",
+              "delete": "Delete Route"
+            },
+            "add-route-dialog": {
+              "title": "Add a Route",
+              "host-label": "Host",
+              "host-error-exists": "A route with this name already exists. Choose a new one",
+              "host-error-required": "Host is required",
+              "host-placeholder": "a unique identifier for this route",
+              "domain-label": "Domain",
+              "route-options-label": "HTTP Route Options",
+              "path-label": "Path (Optional)",
+              "path-placeholder": "optional path for this route",
+              "tcp-label": "TCP Route Options",
+              "port-random-label": "Use random port",
+              "port-label": "Port",
+              "port-placeholder": "port for this route",
+              "button": {
+                "submit": "Create route"
+              }
+            }
+          },
+          "service-instances-panel": {
+            "title": "Service Instances",
+            "manage-button": "Manage Services",
+            "none": "You have no service instances.",
+            "name-label": "Name",
+            "service-label": "Service",
+            "plan-label": "Plan"
+          }
+        },
+        "log-stream": {
+          "label": "Log Stream",
+          "title": "Log Stream",
+          "scroll-button": "Scroll To Bottom"
+        },
+        "services": {
+          "label": "Services",
```

Comment:
Yeah, noticed this a bit too late when I'd already started the label pattern. Didn't quite want to go back and update them all though. I think, going forward, will use the shortened notation. Want me to go back and update these ones?